src/main/java/edu/harvard/iq/dataverse/api/Datasets.java

```diff
@@ -4791,7 +4791,10 @@ public Response deaccessionDataset(@Context ContainerRequestContext crc, @PathPa
                     }
                 }
                 execCommand(new DeaccessionDatasetVersionCommand(req, datasetVersion, false));
-                return ok("Dataset " + datasetId + " deaccessioned for version " + versionId);
+                
+                return ok("Dataset " + 
+                        (":persistentId".equals(datasetId) ? datasetVersion.getDataset().getAuthority() + "/" + datasetVersion.getDataset().getIdentifier() : datasetId) + 
```

Comment:
Won't this be missing "doi:"? What if it's "hdl:" instead?

Can we please have a test?